docs/source/containers/singularity.md

```diff
@@ -0,0 +1,101 @@
+(singularity)=
+# Singularity on Discovery
+
+Singularity is the container runtime engine for the Discovery cluster and is installed as a module. With Singularity, you can run existing containers or build and run your own custom container.
+
+## How to run a container with Singularity
+
+You can run a container with the `run` or `exec` commands after moving to a compute node and loading the singularity module.
+
+To see available singularity versions run: module avail singularity
+
+### The `--bind` or `-B` flag
+
+Its important to mount the directories in discovery to the container so you can access necessary input data and to output the results of your software to the directory you specify. 
+
+### Example using a container file on the file system
```

Comment:
container file has a certain meaning. Normally, Containerfile/Dockerfile so I might change the language to either image file or just say the container on the file system.